addons/sale_timesheet/report/project_profitability_report_analysis.py

```diff
@@ -21,7 +21,9 @@ class ProfitabilityAnalysis(models.Model):
     line_date = fields.Date("Date", readonly=True)
     # cost
     timesheet_unit_amount = fields.Float("Timesheet Duration", digits=(16, 2), readonly=True, group_operator="sum")
+    overtime_unit_amount = fields.Float("Overtime Duration", digits=(16, 2), readonly=True, group_operator="sum")
     timesheet_cost = fields.Float("Timesheet Cost", digits=(16, 2), readonly=True, group_operator="sum")
+    overtime_cost = fields.Float("Overtime Cost", digits=(16, 2), readonly=True, group_operator="sum")
```

Comment:
This could be a monetary field IMO

Reply:
Hello @tivisse, Changes are done. Thanks 😊